src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/Radio.js

```diff
@@ -29,15 +30,26 @@ export default class Radio extends React.PureComponent<Props> {
 
     render() {
         return (
-            <span onClick={this.handleClick} className={radioStyles.radio}>
-                <input
-                    ref={this.setInput}
-                    type="radio"
-                    checked={this.props.checked}
-                    onClick={this.handleInputClick}
-                    onChange={this.handleChange} />
-                <span />
-            </span>
+            <label className={radioStyles.radio}>
+                <span onClick={this.handleClick}>
+                    <input
+                        ref={this.setInput}
+                        type="radio"
+                        checked={this.props.checked}
+                        onClick={this.handleInputClick}
+                        onChange={this.handleChange} />
+                    <span />
+                </span>
+                {this.renderChildren()}
+            </label>
         );
     }
+
+    renderChildren() {
```

Comment:
Is that method really necessary? Doesn't it also work if you simply render `this.props.children` instead?